contracts/RMRK/extension/tokenProperties/RMRKPropertiesAccessControl.sol

```diff
@@ -0,0 +1,251 @@
+// SPDX-LicenseIdentifier: Apache-2.0
+
+pragma solidity ^0.8.18;
+
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+
+import "./IRMRKPropertiesAccessControl.sol";
+
+contract RMRKPropertiesAccessControl is IRMRKPropertiesAccessControl {
+    // mapping(address collection => mapping (ParameterType => mapping (uint256 parameterId => AccessControl))) private _accessControl;
+    mapping(address => mapping(ParameterType => mapping(uint256 => AccessControl)))
+        private _accessControl;
+    // mapping(address collection => IssuerSetting issuerSetting) private _issuerSettings;
```

Comment:
I suspect you're commenting this to be able to run coverage. I solved it on markeplace repo with this: https://github.com/rmrk-team/evm-marketplace/commit/613fd558cf31268b52835fcf625efefeaa955312

Reply:
Amazing!

![image](https://user-images.githubusercontent.com/8097439/229646722-beceae74-a9be-4bfb-9639-e9067ebe348c.png)